medusa/providers/torrent/html/bjshare.py

```diff
@@ -0,0 +1,257 @@
+# coding=utf-8
+
+"""Provider code for BJ-Share."""
+
+from __future__ import unicode_literals
+
+import logging
+import re
+
+from medusa import tv
+from medusa.bs4_parser import BS4Parser
+from medusa.helper.common import convert_size, try_int
+from medusa.logger.adapters.style import BraceAdapter
+from medusa.providers.torrent.torrent_provider import TorrentProvider
+
+from requests.compat import urljoin
+
+log = BraceAdapter(logging.getLogger(__name__))
+log.logger.addHandler(logging.NullHandler())
+
+
+class BJShareProvider(TorrentProvider):
+    """BJ-Share Torrent provider."""
+
+    def __init__(self):
+        """Initialize the class."""
+        super(BJShareProvider, self).__init__('BJ-Share')
+
+        # URLs
+        self.url = 'https://bj-share.me'
+        self.urls = {
+            'search': urljoin(self.url, "torrents.php")
+        }
+
+        # Credentials
+        self.enable_cookies = True
+        self.cookies = ''
+        self.required_cookies = ['session']
```

Comment:
Why did you add this if the provider doesn't require captcha? Why is using user and pass not working?

Reply:
It does require captcha... it's just not always but something trigger it from time to time, i'm not sure what exacly, i had problem with that already, that's why i did with cookies...